jans-cedarling/bindings/cedarling_python/src/config/bootstrap_config.rs

```diff
@@ -5,130 +5,468 @@
+        policy_store_uri: Option<String>,
+        log_type: String,
+        log_ttl: Option<u64>,
+        user_authz: bool,
+        workload_authz: bool,
+        usr_workload_bool_op: String,
+        local_jwks: Option<String>,
+        local_policy_store: Option<String>,
+        policy_store_local_fn: Option<String>,
+        jwt_sig_validation: bool,
+        jwt_status_validation: bool,
+        jwt_signature_algorithms_supported: Vec<String>,
+        at_iss_validation: bool,
+        at_jti_validation: bool,
+        at_nbf_validation: bool,
+        at_exp_validation: bool,
+        idt_iss_validation: bool,
+        idt_sub_validation: bool,
+        idt_exp_validation: bool,
+        idt_iat_validation: bool,
+        idt_aud_validation: bool,
+        userinfo_iss_validation: bool,
+        userinfo_sub_validation: bool,
+        userinfo_aud_validation: bool,
+        userinfo_exp_validation: bool,
+        id_token_trust_mode: String,
+        lock: bool,
+        lock_master_configuration_uri: Option<String>,
+        dynamic_configuration: bool,
+        lock_ssa_jwt: Option<String>,
+        audit_log_interval: u64,
+        audit_health_interval: u64,
+        audit_health_telemetry_interval: u64,
+        listen_sse: bool,
     ) -> PyResult<Self> {
-        let log_config = match log_config {
-            Some(python_value) => Some(extract_log_config(&python_value)?),
-            None => None,
-        };
-
         Ok(Self {
             application_name,
-            policy_store_config,
-            log_config,
-            jwt_config,
+            policy_store_uri,
+            policy_store_id,
+            log_type,
+            log_ttl,
+            user_authz,
+            workload_authz,
+            usr_workload_bool_op,
+            local_jwks,
+            local_policy_store,
+            policy_store_local_fn,
+            jwt_sig_validation,
+            jwt_status_validation,
+            jwt_signature_algorithms_supported,
+            at_iss_validation,
+            at_jti_validation,
+            at_nbf_validation,
+            at_exp_validation,
+            idt_iss_validation,
+            idt_sub_validation,
+            idt_exp_validation,
+            idt_iat_validation,
+            idt_aud_validation,
+            userinfo_iss_validation,
+            userinfo_sub_validation,
+            userinfo_aud_validation,
+            userinfo_exp_validation,
+            id_token_trust_mode,
+            lock,
+            lock_master_configuration_uri,
+            dynamic_configuration,
+            lock_ssa_jwt,
+            audit_log_interval,
+            audit_health_interval,
+            audit_health_telemetry_interval,
+            listen_sse,
         })
     }
 
-    #[setter]
-    fn log_config(&mut self, value: PyObject) -> PyResult<()> {
-        self.log_config = Some(extract_log_config(&value)?);
-        Ok(())
+    pub fn disable_all_jwt_validation(&mut self) -> () {
```

Comment:
Maybe better implement this method for appropriate kind of structure in cedarlind (core)? To allow use it in other bindings?

Reply:
That's a good idea. Perhaps this could be passed in the authz request... for example in a `AuthzRequestOptions` object?